### ISSUE TITLE
Train Data No Longer Spawns

### ISSUE THREAD

**Issue by halosammy** [CleverRaven/Cataclysm-DDA#57804]
### Describe the bug

After a considerable amount of time verifying if something was, indeed, actually wrong with the game, it seems as if the mission known as “Find Lab Tunnels” is no longer completable. Not only does the mission waypoint direct you to a microlab, which only comes with station instead of a depot, the terminals in all the lab depots you _can_ find offer no option to download the data you need.

### Steps to reproduce

1. Acquire “Find Lab Tunnels” from the refugee center doctor.
2. Follow waypoint to the microlab.
3. Search for depots using debug teleporter.

### Expected behavior

The central terminal of any given lab depot would have an option to download the quest data to an empty USB.

### Versions and configuration

0a6883a

**Comment by PatrikLundell** [comment]
/confirmed

My current game directed me to a lab under a research facility, but the lab didn't have any subway station access at all. At Z-2 there was nothing, and it didn't reach Z-4 (and there was a finale at Z-3). There also wasn't any obvious access from a "nearby" lab that did have a Z-4 subway access: that track didn't get anywhere near the science lab location (for as far as I followed it). At a guess, the quest was taken two weeks ago or so, on a reasonably up to date experimental version.

I encountered a similar failure in my previous game (E-2, I think), so the problem is an old one. If I remember correctly, the railway track passed that lab at a distance of two tiles or so, but there was no subway station (the track was at the right depth, Z-4).
